internal/impl/otlp/tracer_otlp.go

```diff
@@ -122,6 +123,14 @@ func newOtlp(config *otlp) (trace.TracerProvider, error) {
 		attrs = append(attrs, attribute.String(k, v))
 	}
 
+	if _, ok := config.tags[string(semconv.ServiceNameKey)]; !ok {
+		attrs = append(attrs, semconv.ServiceNameKey.String("benthos"))
+	}
+
+	if _, ok := config.tags[string(semconv.ServiceVersionKey)]; !ok {
+		attrs = append(attrs, semconv.ServiceVersionKey.String(cli.Version))
+	}
```

Comment:
I think the default version should only be set if the default service name is being set. This would cover situations where user's only set service name.

Also, does it work to pass these as default tags so they're reflected in the docs? 👇 

```
		Field(service.NewStringMapField("tags").
			Description("A map of tags to add to all tracing spans.").
			Default(map[string]string{
				string(semconv.ServiceNameKey): "benthos",
				string(semconv.ServiceVersionKey): cli.Version,
            }).
Advanced())
```

Reply:
Having them as an explicit config default gets a bit weird, it'd mean the docs change after every tagged release, also means setting any tag would wipe away the defaults which we might not want.

Reply:
@disintegrator Given Ash's remark, I addressed this part: "they should accidentally get the benthos version as an additional tag". Guess you meant "shouldn't".

I also changed the logic for the Jaeger tracer, which I guess is a bit of a breaking change, but I doubt anyone will get tripped up by that...

Reply:
@mihaitodor yep that’s a typo 🤦 
Fixed…